providers/microsoft/azure/pyproject.toml

```diff
@@ -120,6 +120,7 @@ dev = [
     "apache-airflow-providers-sftp",
     # Additional devel dependencies (do not remove this line and add extra development dependencies)
     "pywinrm>=0.4",
+    "setuptools>=76.0.0,!=77.0.1",
```

Comment:
This isn't a dev dep is it -- from the stack trace it looked like it needs this at runtime.

Reply:
Possibly guarded behind a `; python_version >= "3.12"` too?